Makefile

```diff
@@ -269,7 +269,7 @@ clean-easyjson-srcs:
 .PHONY: go-generated-srcs
 go-generated-srcs: deps clean-easyjson-srcs go-easyjson-srcs $(MOCKGEN_BIN) $(STRINGER_BIN) $(GENNY_BIN)
 	@echo "+ $@"
-	PATH="$(PATH):$(BASE_DIR)/tools/generate-helpers" go generate -v -x ./...
+	PATH="$(PATH):$(BASE_DIR)/tools/generate-helpers" ROX_VERIFY_IMAGE_SIGNATURE=true go generate -v -x ./...
```

Comment:
Why turn the flag on here?

Reply:
Good question. The resource is behind feature flag so when I try to generate code for it I cannot get it's metadata unless I set this flag. I'm not sure how to solve this issue. Another way will be to have a list of all resources (including those behind feature flag).

Reply:
Maybe having a method `resources.ListAllDisabledMetadata()` will be cleaner and future proof